requirements/dev.in

```diff
@@ -0,0 +1,9 @@
+# Additional requirements for development of this application
+-c constraints.txt
+
+-r base.in                # Dependencies for edx-completion itself
```

Comment:
This should be base.txt in order to use the same dependencies generated by base.in in order to prevent the possiblity of different release if there is a new release just in the moment that this file is being compiled.

In general, I think that we can use just test.txt and quality.txt

Reply:
``dev.in`` roles were given to ``base.in``.
deleted ``dev.in``.